web/viewer-geckoview.js

```diff
@@ -45,6 +45,7 @@ function getViewerConfiguration() {
       mainContainer,
       container: document.getElementById("floatingToolbar"),
       download: document.getElementById("download"),
+      openInApp: document.getElementById("openinapp"),
```

Comment:
Can we please also capitalize the DOM-element name *itself* to aid readability?
```suggestion
      openInApp: document.getElementById("openInApp"),
```